frontend/app/views/spree/products/_filters_desktop.html.erb

```diff
@@ -56,6 +56,9 @@
           </div>
         </div>
       </div>
+      <% additional_filters_partials.each do |partial| %>
+        <%= render partial, permitted_params: permitted_params %>
+      <% end %>
```

Comment:
```suggestion
      <% filters_partials.each do |partial| %>
        <%= render partial, permitted_params: permitted_params %>
      <% end %>
```
would be even better if you are willing to work on moving default filters to partials